appscale/tools/parse_args.py

```diff
@@ -361,6 +361,13 @@ def add_allowed_flags(self, function):
         help="the name of the AppController property to set")
       self.parser.add_argument('--property_value',
         help="the value of the AppController property to set")
+    elif function == "appscale-upgrade":
+      self.parser.add_argument('--keyname', '-k', default=self.DEFAULT_KEYNAME,
+        help="the keypair name to use")
+      self.parser.add_argument('--ips_layout',
+        help="a YAML file dictating the placement strategy")
+      self.parser.add_argument('--unique_ips',
```

Comment:
Does this command need a list of unique IP addresses if it already has the full ips_layout?